lib/screens/esummit/esummit.dart

```diff
@@ -0,0 +1,122 @@
+            child: Container(
+                decoration: BoxDecoration(
+                    gradient: LinearGradient(
+                  begin: Alignment.topCenter,
+                  end: Alignment.bottomCenter,
+                  stops: [0.3, 1],
+                  colors: [C.backgroundTop1, C.backgroundBottom1],
+                )),
+                child: Column(mainAxisSize: MainAxisSize.min, children: [
+                  SizedBox(height: height / 6),
+                  Expanded(
+                      flex: 3,
+                      child: Column(children: [
+                        Container(
+                            alignment: Alignment.centerLeft,
+                            padding: EdgeInsets.only(left: D.horizontalPadding + 1),
+                            child: Image.asset(S.assetEsummitLogoWhite,
+                                width: width * 0.3 * heightFactor)),
+                        Container(
+                            alignment: Alignment.centerLeft,
+                            padding: EdgeInsets.only(left: D.horizontalPadding, top: 20),
+                            child: RichText(
+                                text: TextSpan(
+                                    text: "E-Summit '21",
+                                    style: TextStyle(
+                                        fontSize: 45 * heightFactor,
+                                        fontWeight: FontWeight.w900)))),
+                        Container(
+                            alignment: Alignment.centerLeft,
+                            padding: EdgeInsets.only(left: D.horizontalPadding, top: 30),
+                            child: RichText(
+                                text: TextSpan(
+                                    text: "The Genesis of Innovation",
+                                    style: TextStyle(
+                                        fontSize: 30 * heightFactor,
+                                        fontWeight: FontWeight.bold)))),
+                        Container(
+                            alignment: Alignment.centerLeft,
+                            padding:
+                                EdgeInsets.symmetric(horizontal: D.horizontalPadding, vertical: 30),
+                            child: RichText(
+                                text: TextSpan(
+                                    text: "At ",
+                                    style: TextStyle(
+                                        fontSize: 20 * heightFactor,
+                                        fontWeight: FontWeight.w300,
+                                        height: 1.2,
+                                        wordSpacing: 4),
+                                    children: <TextSpan>[
+                                  TextSpan(
+                                      text: "E-Summit ",
+                                      style: TextStyle(
+                                          color: C.speakerButtonColor,
```

Comment:
please coordinate with mohit about colors. I had a talk and he told me color here is different from the button color. `#5D02A3`

Reply:
as asked from mohit he told to use #8E31D5 for both the speaker button and the highlighted text.

Reply:
Coordinate with mohit. He''ll update you.

Reply:
i had a talk with him just after your review he said to use the #8E31D5 color for both.